README.md

```diff
@@ -321,15 +344,43 @@ Take your time to plan out your code, test each method, and make many commits. A
 
 ### Phase 4: Project Review
 
-When you are ready, schedule a project review with one of your instructors. The format of the review will roughly look like this:
+Communicate with your instructor to schedule a project review. The format of the review will roughly look like this:
 
-* 3 minutes - Requirements Review
+* 5 minutes - Requirements Review
 * 5 minutes - App Overview
-* 6 minutes - Deep Dive
-* 12 minutes - Pair Programming
-* 4 minutes - Feedback
+* 15 minutes - Deep Dive
+* 15 minutes - Pair Programming
+* 5 minutes - Wrap up
+
+In the interview, the instructor will check that your project meets the deliverables, have you explain your code, and check your understanding of the concepts through pair programming. 
+
+They'll use the following rubric to evaluate the project and interview.
+
+#### Rubric
+
+##### Designing a CLI Application
+
+1. Incomplete or nonfunctioning CLI. Student is not able to explain how the code is loaded or how the classes interact with each other.
+2. CLI is functioning, but code is scrambled. Student can explain some of how the code works, but has gaps in their explanation.
+3. CLI has least 4 user stories, and the CLI interface is separated from the models in the domain. The student can explain how the code works, but not why the code was architected the way it was. Code may be factored poorly, and might not handle errors or handle unexpected user input.
```

Comment:
I'm thinking very defensively for the small number of edge-cases where we face a lot of student resistance, but maybe we want to at least somewhat more specific than "4 user stories", even if the language was vague but gave teachers more weight to back up decisions. What about "4 user stories that meet complexity requirements in guidelines", or "4 user stories tied to different CRUD actions"

Reply:
```suggestion
3. CLI has 4 user stories that meet the complexity requirements in the instructions, and the CLI interface is separated from the models in the domain. The student can explain how the code works, but not why the code was architected the way it was. Code may be factored poorly, and might not handle errors or handle unexpected user input.
```